nautobot/docs/development/index.md

```diff
@@ -164,7 +155,11 @@ categories for discussions:
   work is required to resolve them. It may take some time for someone to address
   your issue.
 
-#### Feature Requests
+Bugs will follow our published workflow from inbound triage to ultimate terminal state, whether accepted or closed:
+
+![Bug Workflow State Diagram](imgs/bug-wf.jpg)
```

Comment:
Instead of creating another new directory for doc images (we already have `docs/img/`, `docs/media/` and `docs/user-guides/images/`) can these images be added to the existing `docs/media/development/` directory instead?

Reply:
Moved!